src/mutations/draft/putDraft.ts

```diff
@@ -33,14 +33,7 @@ const resolver: MutationToPutDraftResolver = async (
   { input },
   { viewer, dataSources: { draftService, systemService, articleService } }
 ) => {
-  const {
-    id,
-    title,
-    content,
-    tags,
-    coverAssetId: coverAssetUUID,
-    collection: collectionGlobalIds,
-  } = input
+  const { id, title, content, tags, cover, collection: collection } = input
```

Comment:
Can be short-handed as `const { id, title, content, tags, cover, collection } = input`?